<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>361</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Смяна на Admin парола</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Текуща парола</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCurrentPass">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Нова парола</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNewPass">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNewPassRepeat">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Повторете парола</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSavePass">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Запис</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
